src/slider/Slider.jsx

```diff
@@ -135,16 +135,23 @@ class Slider extends React.Component {
     };
 
     onKeyDown = (event) => {
-        const { max, min } = this.props;
+        event.preventDefault();
+        const { disabled, max, min } = this.props;
         const handler = this.keyBinding[event.keyCode];
 
-        if (handler) {
+        if (handler && !disabled) {
             let value = handler(this.props);
             value = util.trimValue(max, min, value);
             this.props.onChange({ value });
         }
     };
 
+    onComponentClick = () => {
+        if (!this.props.disabled) {
+            ReactDOM.findDOMNode(this).getElementsByClassName('k-draghandle')[0].focus();
```

Comment:
You could use `this.componentElements().dragHandle` to get reference to the drag handle.